### DISCUSSION TITLE
appstore_sandbox config setting

### DISCUSSION THREAD

**Discussion by iammart** [imdhemy/laravel-in-app-purchases#61] [Q&A]
Hi there,

Im currently using 0.9.0, I see you made an update in 0.8.5 that removes the key/value `appstore_sandbox` from the purchase config.

A project I was working on went live with this still configured in the config with a value of true

We are receiving a lot of errors in logs identical to the following:

```
production.ERROR: This receipt is from the production environment, but it was sent to the test environment for verification. {"exception":"[object] (Imdhemy\\AppStore\\Exceptions\\InvalidReceiptException(code: 21008): This receipt is from the production environment, but it was sent to the test environment for verification. at /home/xxxxx/vendor/imdhemy/appstore-iap/src/Exceptions/InvalidReceiptException.php:38)
```

I understood the change would first hit the production URL and then fallback to sandbox, even though it was removed from readme it still exists

I doesn't appear to be causing issues as we have subscriptions renewing fine. However, moving forward would me changing the value to false or removing it(the default is currently an empty string) interfere with anything?

Thanks in advance

**Comment by imdhemy** [comment]
Hi @iammart,

At first, I highly recommend you upgrade to the latest version as it has some fixes and doesn't break the old code.

Feel free to remove the `appstore_sandbox` configuration as it doesn't interfere with anything.

**Reply by iammart** [reply]
Ok, thank you. I will look into this
